index.bs

```diff
@@ -12516,6 +12558,10 @@ The characteristics of a namespace object are described in [[#namespace-object]]
         1.  Let |F| be the result of [=creating an operation function|creating an operation function=]
             given |op|, |namespace|,  and |realm|.
         1.  Perform [=!=] [=CreateDataProperty=](|namespaceObject|, |op|'s [=identifier=], |F|).
+    1.  For each [=exposed=] [=interface=] |intr| which has the [{{InNamespace}}] extended attribute, with the identifier of this namespace as its argument,
```

Comment:
Wrap to 100 chars here too

`[=identifier=]`

Just _interface_ instead of _intr_ is nicer, I think.

No comma before "with", I think.

Reply:
Done